pkg/resource/deploy/step.go

```diff
@@ -326,7 +326,10 @@ func (s *DeleteStep) Apply(preview bool) (resource.Status, StepCompleteFunc, err
 	// Refuse to delete protected resources.
 	if s.old.Protect {
 		return resource.StatusOK, nil,
-			errors.Errorf("refusing to delete protected resource '%s'", s.old.URN)
+			errors.Errorf("unable to delete resource %q\n"+
+				"as it is currently marked for protection. To unprotect the resource, "+
+				"either remove the `protect` flag from the resource in your Pulumi application or use the command:\n"+
```

Comment:
To align with documented terminology (https://www.pulumi.com/docs/intro/concepts/):

```suggestion
				"either remove the `protect` flag from the resource in your Pulumi program or use the command:\n"+
```